origin-bridge/dev.env

```diff
@@ -29,6 +29,7 @@ TWITTER_CONSUMER_KEY=
 TWITTER_CONSUMER_SECRET=
 
 ATTESTATION_SIGNING_KEY=
+ATTESTATION_ACCOUNT
```

Comment:
I'm not sure of the consequences of leaving off the `=` here, it might be nothing but it still might be good to add it.

Reply:
Good eye. Fixed.